lib/renderer/override.js

```diff
@@ -4,5 +4,6 @@ const {ipcRenderer} = require('electron')
 
 const {guestInstanceId, openerId} = process
 const hiddenPage = process.argv.includes('--hidden-page')
+const usesNativeWindowOpen = process.argv.includes('--native-window-open')
```

Comment:
I've updated it in https://github.com/electron/electron/pull/8963/commits/8b6b512485a2928f6e93630889b530986f6bf428.